command/ca/provisioner/getEncryptedKey.go

```diff
@@ -5,18 +5,18 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/smallstep/certificates/pki"
-	"github.com/smallstep/cli/errs"
 	"github.com/smallstep/cli/flags"
 	"github.com/urfave/cli"
+	"go.step.sm/cli-utils/errs"
 )
 
 func getEncryptedKeyCommand() cli.Command {
 	return cli.Command{
 		Name:   "jwe-key",
 		Action: cli.ActionFunc(getEncryptedKeyAction),
 		Usage:  "retrieve and print a provisioning key in the CA",
-		UsageText: `**step ca provisioner jwe-key** <kid> 
-[**--ca-url**=<uri>] [**--root**=<file>]`,
+		UsageText: `**step ca provisioner jwe-key** <name|kid>
```

Comment:
What I meant is to implement it, if it's not yet, just leave it as `<kid>`

Reply:
Ah, sorry.